creator-node/src/serviceRegistry.js

```diff
@@ -25,45 +25,42 @@ const PrometheusRegistry = require('./services/prometheusMonitoring/prometheusRe
- *  - `monitoringQueue`: recurring job to monitor node state & performance metrics
- *  - `sessionExpirationQueue`: recurring job to clear expired session tokens from Redis and DB
- *  - `asyncProcessingQueue`: queue that processes jobs asynchronously and adds job responses into redis
- *
- *  - `libs`: an instance of Audius Libs
- *  - `snapbackSM`: SnapbackStateMachine is responsible for recurring sync and reconfig operations
- *  - `URSMRegistrationManager`: registers node on L2 URSM contract, no-ops afterward
- *
- * `initServices` must be called prior to consuming services from the registry.
  */
 class ServiceRegistry {
   constructor() {
-    this.nodeConfig = config
-    this.redis = redisClient
-    this.blacklistManager = BlacklistManager
-    this.monitoringQueue = new MonitoringQueue()
-    this.sessionExpirationQueue = new SessionExpirationQueue()
-    this.prometheusRegistry = new PrometheusRegistry()
-
-    // below services are initialized separately in below functions `initServices()` and `initServicesThatRequireServer()`
-    this.libs = null
-    this.stateMachineManager = null
-    this.snapbackSM = null
-    this.URSMRegistrationManager = null
-    this.syncQueue = null
-    this.skippedCIDsRetryQueue = null
-    this.trustedNotifierManager = null
-
-    this.servicesInitialized = false
+    // Some services are initialized to `null` and will be initialized in helper functions
+
+    this.redis = redisClient // Redis Client
+    this.prometheusRegistry = new PrometheusRegistry() // Service that tracks metrics
+    this.libs = null // instance of Audius Libs
+    this.blacklistManager = BlacklistManager // Service that handles blacklisted content
+    this.stateMachineManager = null // Service that manages user states
+    this.snapbackSM = null // Responsible for recurring sync and reconfig operations
+    this.URSMRegistrationManager = null // Registers node on L2 URSM contract, no-ops afterward
+    this.trustedNotifierManager = null // Service that blacklists content on behalf of Content Nodes
+
+    // Queues
+    this.monitoringQueue = new MonitoringQueue() // Recurring job to monitor node state & performance metrics
+    this.sessionExpirationQueue = new SessionExpirationQueue() // Recurring job to clear expired session tokens from Redis and DB
+    this.imageProcessingQueue = ImageProcessingQueue // Resizes all images on Audius
+    this.transcodingQueue = TranscodingQueue // Transcodesa and segments all tracks
+    this.skippedCIDsRetryQueue = null // Retries syncing CIDs that were unable to sync on first try
+    this.syncQueue = null // Handles syncing data to users' replica sets
+    this.asyncProcessingQueue = null // Handles all jobs that should be performed asynchornously. Currently handles track upload and track hand off
+    this.stateMonitoringQueue = null // ask theo
+    this.stateReconciliationQueue = null // ask theo
+    this.stateMachineQueue = null // ask theo
```

Comment:
stateMonitoringQueue // Handles jobs for finding replica set updates and syncs for one slice of users at a time

stateReconciliationQueue // Handles jobs for issuing sync requests and updating users' replica sets

stateMachineQueue // DEPRECATED -- being removed very soon